### ISSUE TITLE
Log status should be updated across offline and online lists

### ISSUE THREAD

**Issue by cachly** [cachly/issues-tracker#4914]
**[Original report](https://bitbucket.org/zedsaidstudio/public-issues-tracker/issues/54) by Thomas Grehl (Bitbucket: [ThomGr](https://bitbucket.org/ThomGr), GitHub: [ThomGr](https://github.com/ThomGr)).**

----------------------------------------

When a cache is present in offline lists, PQs (offline and online) and the search results it should show the same log status (found, DNF, field note written) no matter where the log was initiated. Example: log from search results a find, where the cache is also in any of the offline lists

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

Not following you here. Can you explain a bit more?

**Comment by cachly** [comment]
**Original comment by Thomas Grehl (Bitbucket: [ThomGr](https://bitbucket.org/ThomGr), GitHub: [ThomGr](https://github.com/ThomGr)).**

----------------------------------------

I guess that between offline lists, the status of the cache is already in sync as the cache is only stored once even if it exists in multiple lists. However, although I did not test is recently, if I log a cache in online mode (or send a field note), the status should also be updated for the same cache should it exist in any offline list. 
Sounds unimportant, but here is my usecase: I use the offline lists when abroad, but may have coverage in some areas so might as well use the live view. So when I log from there, I want to see later on which caches I found in the offline list without refreshing them.
